src/HPND-sell-variant-MIT-disclaimer-rev.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false"
+   licenseId="HPND-sell-variant-MIT-disclaimer-rev" name="HPND sell
+   variant with MIT disclaimer - reverse" listVersionAdded="3.24">

```

Comment:
There cannot be a linebreak in the value of attribute. This is XML thing.
Add linebreak before "name" and before "listVersionAdded" and it will be good.